deploy.ps1

```diff
@@ -0,0 +1,59 @@
+# PowerShell script to deploy the main.bicep template with parameter values
+
+#Requires -Modules "Az"
+#Requires -PSEdition Core
+
+# Use these parameters to customize the deployment instead of modifying the default parameter values
+[CmdletBinding()]
+Param(
+    [ValidateSet('eastus2', 'eastus')]
+    [Parameter()]
+    [string]$Location = 'eastus',
```

Comment:
If we are thinking of sharing this script to customers, how about we change this a required parameter (i.e., no default value) and remove ValidateSet? Or we may get the input from user with an expected values like?

```
$location = Read-Host -Prompt "Provide the location (like, eastus, westus, eastus2, westus2,etc)?"
```